lumibot/strategies/strategy_executor.py

```diff
@@ -184,6 +199,11 @@ def add_event(self, event_name, payload):
         self.queue.put((event_name, payload))
 
     def process_event(self, event, payload):
+        # If it's the first iteration, we don't want to process any events.
+        # This is because in this case we are most likely processing events that occurred before the strategy started.
+        if self.strategy._first_iteration or self.broker._first_iteration:
+            return
+
```

Comment:
I'm not sure this is quite right.  This prevents all Polling() operations from being processed until after the first iteration has completed.  I'm imaging a situation where you have:
- long iterations set (15m or larger)
- a long multi-day position with a stoploss
- stoploss gets triggered just after market open

With this update, the StopLoss fill cannot be processed until a later iteration - potentially much later.  Maybe instead of using a "first_iteration" check, there should be a separate flag that is "first_broker_sync" instead because it can be cleared much faster than the first_iteration.

Reply:
why would market open matter here? first iteration is the first iteration ever for the algorithm, not per day. eg. if i ran the algo right now it would flip first iteration to False and it will stay False for the remainder of its life.